src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewCell.swift

```diff
@@ -133,7 +133,7 @@ extension DynamicCell {
 		.identifier(CellReuseIdentifier.space) { _, cell, _ in
 			guard let cell = cell as? DynamicTableViewSpaceCell else { return }
 			cell.height = height
-			cell.backgroundColor = color
+			cell.backgroundColor = .clear
```

Comment:
Would be a better idea to leave it as it was and change the function to `static func space(height: CGFloat, color: UIColor = .clear) -> Self {`

Reply:
Thanx! Done.